scheds/rust/scx_layered/src/main.rs

```diff
@@ -2315,6 +2312,12 @@ fn main() -> Result<()> {
         );
     }
 
+    for spec in &layer_config.specs {
```

Comment:
Maybe we could have a macro for deprecating fields to generate these warnings, could be a good help wanted task.